src/Raven.Server/Documents/ShardedHandlers/ShardedStatsHandler.cs

```diff
@@ -28,6 +31,33 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        private IndexInformation[] GetDatabaseIndexesFromRecord()
+        {
+            var record = ShardedContext.DatabaseRecord;
+            var indexes = record.Indexes;
+            var indexInformation = new IndexInformation[indexes.Count];
+
+            int i = 0;
+            foreach (var key in indexes.Keys)
+            {
+                var index = indexes[key];
+
+                indexInformation[i] = new IndexInformation
+                {
+                    State = index.State ?? IndexState.Error,
+                    Name = index.Name,
+                    LockMode = index.LockMode ?? IndexLockMode.LockedError,
+                    Priority = index.Priority ?? new IndexPriority(),
```

Comment:
`IndexPriority` is an `enum` so it is wired for me to see `new IndexPriority()`.
Does the `Priority`/`LockMode`/`State` are nullable?

Wonder what about `IsStale`.. not sure what do to with that, but let's set it explicitly here.

Also the `State` is from the index definition while the actual state can be different. (e.g. we can disable an index locally)

We don't want to handle this right now, but do need to keep that in mind